### PR TITLE
Add fix for 'too many open files' to README

### PR DESCRIPTION
**Pull request by awiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,8 @@ $ git clone https://github.com/grailbio/bigslice
 $ cd bigslice
 $ GO111MODULE=on go test
 ```
+
+If you experience errors running tests, and see `too many open files` in the logs, you can increase the maximum number of open files by running
+```
+ulimit -n 2000
```
Comment (jcharum) [inline README.md:31]:
```suggestion
$ ulimit -n 2000
```

File: README.md [hunk 2]
```diff
@@ -25,3 +25,8 @@ $ git clone https://github.com/grailbio/bigslice
 $ cd bigslice
 $ GO111MODULE=on go test
 ```
+
+If you experience errors running tests, and see `too many open files` in the logs, you can increase the maximum number of open files by running
```
Comment (jcharum) [inline README.md:29]:
```suggestion
If tests fail with `socket: too many open files` errors, try increasing the maximum number of open file descriptors:
```

Comment (jcharum) [reply README.md:29]: Save some words.

File: README.md (hunk 1 above)
Comment (jcharum) [reply README.md:31]: Make invocation consistent with above.

**Comment by awiss** [comment]
Thanks for the comments, much better.

File: README.md [hunk 3]
```diff
@@ -25,3 +25,8 @@ $ git clone https://github.com/grailbio/bigslice
 $ cd bigslice
 $ GO111MODULE=on go test
 ```
+
+If tests fail with `socket: too many open files` errors, try increasing the maximum number of open files by running
```
Comment (jcharum) [inline README.md:29]:
```suggestion
If tests fail with `socket: too many open files` errors, try increasing the maximum number of open files.
```

Comment (jcharum) [reply README.md:29]: FYI, you can directly commit "suggestions" by clicking the `Commit suggestion` button. This basically saves the work of copying over suggestions yourself. (It's one of my favorite features of GitHub's code reviews.)

**Comment by jcharum** [comment]
I'm not sure why the (recently set-up) GitHub actions lint is failing. It is a known problem that because of a malformed `go.mod` in one of the dependencies, `golangci-lint` will fail if run with Go 1.14, as it is more strict about its `go.mod` checking than Go 1.13. However, I thought that was (deterministically) fixed by installing Go 1.13 beforehand. That appears to not be the case. I will restart the jobs and separately look into fixing the lint to not be non-deterministic in this way (unless that's something you want to do).

**Comment by awiss** [comment]
How do you restart these builds? I was looking around but couldn't find anything in the UI.

**Comment by jcharum** [comment]
If you click on `Details`, you will be taken to a page with the workflows. There should be a button on the right to restart all jobs.

**Pull request merged**